app/assets/javascripts/discourse/app/components/sidebar/api-section.js

```diff
@@ -27,8 +27,14 @@ export default class SidebarApiSection extends Component {
     if (this.section.text.toLowerCase().match(this.sidebarState.filter)) {
       return this.section.links;
     }
+
     return this.section.links.filter((link) => {
-      return link.text.toString().toLowerCase().match(this.sidebarState.filter);
+      return (
+        link.text.toString().toLowerCase().match(this.sidebarState.filter) ||
+        link.keywords?.some((keyword) =>
```

Comment:
I think you don't need the `?.` safe nav, because `keywords` is just `[]` if none are provided